storage/storage.go

```diff
@@ -200,28 +207,18 @@ type hybridClientOptions struct {
 	GRPCOpts []option.ClientOption
 }
 
-// newHybridClient creates a new Storage client that initializes a gRPC-based client
-// for media upload and download operations.
+// newGRPCClient creates a new Storage client that initializes a gRPC-based
+// client. Calls that have not been implemented in gRPC will panic.
 //
 // This is an experimental API and not intended for public use.
-func newHybridClient(ctx context.Context, opts *hybridClientOptions) (*Client, error) {
```

Comment:
Can you also remove the `type hybridClientOptions struct`? Or is that being used somewhere still?

Reply:
Good point, I missed this! Will remove as well.